src/apps/statements/tests/getStatements/cursor-same-stored-values.test.ts

```diff
@@ -0,0 +1,89 @@
+import connectToMongoDb from '../../repo/utils/connectToMongoDb';
+import createClientModel from '../utils/createClientModel';
+import createStatement from '../utils/createStatement';
+import setup from '../utils/setup';
+import storeStatementsInService from '../utils/storeStatementsInService';
+import { assertStatementsPageResultsAndOrder } from './utils/assertStatementsPageResultsAndOrder';
+
+const TEST_ID_1 = '46bca0ac-426f-44ec-8b7d-04eb31dd1f22';
+const TEST_ID_2 = '92718e8a-8eaf-4f20-b347-0a82cdedacf0';
+const TEST_ID_3 = '4a39fc67-eab0-4b2e-a8fd-5785d7600160';
+const TEST_ID_4 = 'cf3b2ddd-ae7d-4f48-b653-02a385f64067';
+const TEST_ID_5 = '3418987e-fc23-497f-9e3c-cb00d2783255';
+
+const TEST_CLIENT = createClientModel();
+
+describe('get statements with the same stored value using cursor', async () => {
+  const service = setup();
+  const db = await connectToMongoDb()();
+  const storeStatements = storeStatementsInService(service);
+
+  const statement1 = createStatement({ id: TEST_ID_1 });
+  const statement2 = createStatement({ id: TEST_ID_2 });
+  const statement3 = createStatement({ id: TEST_ID_3 });
+  const statement4 = createStatement({ id: TEST_ID_4 });
+  const statement5 = createStatement({ id: TEST_ID_5 });
+
+  beforeEach(async () => {
+    await db.dropDatabase();
```

Comment:
@ryansmith94 , I'm not sure that this is ok, but without it tests gonna fail in `cover-ci` mode. I guess database doesn't get cleared at some point. Could you please confirm it or provide better way to do it? Also there is a same thing in another similar test pack

Reply:
I think this is taken care of in the `setup` function called above.

Reply:
But as I said my tests were failed without it because of some data from previous tests

Reply:
Interesting. I thought the setup function had a beforeEach that cleared up between all tests.